src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -225,9 +229,7 @@ protected override IAttributeTargetSymbol AttributeOwner
         }
 
         internal override bool IsExpressionBodied
-        {
-            get { return false; }
-        }
+            => _isExpressionBodied;
```

Comment:
Either put this on the previous line, or (preferred) follow the style used in this file and use block as a body.